srcpkgs/freerdp3/template

```diff
@@ -0,0 +1,119 @@
+# Template file for 'freerdp3'
+pkgname=freerdp3
+version=3.5.0
+revision=1
+build_style=cmake
+build_helper=qemu
+configure_args="
+ -DRDTK_FORCE_STATIC_BUILD=ON
+ -DUWAC_FORCE_STATIC_BUILD=ON
+ -DWITH_BINARY_VERSIONING=ON
+
+ -DCHANNEL_URBDRC_CLIENT=ON
+ -DDOCBOOKXSL_DIR=/usr/share/xsl/docbook
+ -DPROXY_PLUGINDIR=/usr/lib/${pkgname}/server/proxy/plugins
+ -DWAYLAND_SCANNER=/usr/bin/wayland-scanner
+ -DXSLTPROC_EXECUTABLE=/usr/bin/xsltproc
+
+ -DPKG_CONFIG_RELOCATABLE=NO
+ -DWITH_ALSA=ON
+ -DWITH_CAIRO=ON
+ -DWITH_CHANNELS=ON
+ -DWITH_CLIENT_CHANNELS=ON
+ -DWITH_CLIENT_SDL=ON
```

Comment:
This either has to be dropped, or `makedepends` needs `SDL2-devel`, `SDL2_ttf-devel` and `webkit2gtk-devel`, although I can't get `sdl-freerdp3` to actually show me a remote window.

Reply:
enabled sdl client without webview.